apps/site/assets/ts/schedule/components/schedule-finder/UpcomingDepartures.tsx

```diff
@@ -48,17 +55,28 @@ export const RoutePillSmall = ({
     </span>
   </span>
 );
-interface TableRowProps {
-  input: UserInput;
-  journey: EnhancedJourney;
-}
 
-const BusTableRow = ({
+export const BusTableRow = ({
   journey
 }: {
   journey: EnhancedJourney;
 }): ReactElement<HTMLElement> | null => {
-  const { trip, route, realtime } = journey;
+  const { trip, route, realtime, tripInfo } = journey;
+
+  // Only display the crowding information if the trip ID of the vehicle matches the trip ID of the prediction being displayed.
+  let showCrowding = false;
+  let crowding = "crowding_unavailable";
+  let crowdingText = "";
+
+  if (tripInfo.vehicle) {
+    showCrowding = tripInfo.vehicle.trip_id === trip.id;
```

Comment:
I'm wondering if it's possible to simplify these conditionals and ideally not have to use `let`s. Below there's another check for `showCrowding && crowdingText !== ""`. Am I correct in thinking that `crowdingText` would be `""` in the case where the `vehicle` in `tripInfo` has `null` for `crowding`? If I've got that right, would it be possible to change this line to be something along the lines of, `showCrowding = tripInfo.vehicle.trip_id === trip.id && tripInfo.vehicle.crowding;`? That might allow you to just use `crowdingDescriptions` safely too?